client/src/review/productBreakdown.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 
 const ProductBreakdown = (props) => (
-  <div>
+  <div style={{ width: '250px', marginLeft: '50px' }}>
```

Comment:
Just making sure you meant to add some in-line stying here

Reply:
yes, I used some inline styling somewhere (especially for some nonrepeated items). Not sure if this is the brest practice. If not I will move everything to the app.css.